yk_face/face.py

```
@@ -96,6 +97,7 @@ async def process_async(
     :raises:
         ValueError if image is not provided.
     """
+    configurations = configurations or []
```

We should also raise an exception if processings is empty or null.